doc/tutorials/basic/index.md

```diff
@@ -0,0 +1,332 @@
+:::
+
+:::{grid-item-card} {octicon}`arrow-switch;2.5em;sd-mr-1` React to User Input
+:link: pn_bind
+:link-type: doc
+
+React to user input by leveraging `pn.bind`, enabling our users to foster a deeper understanding of wind turbine data through exploration.
+:::
+
+:::{grid-item-card} {octicon}`list-ordered;2.5em;sd-mr-1` Handle State
+:link: state
+:link-type: doc
+
+Be able to build larger and more complex apps by defining and maintaining state via `pn.rx`.
+:::
+
+:::{grid-item-card} {octicon}`checklist;2.5em;sd-mr-1` Build a Todo App
+:link: build_todo
+:link-type: doc
+
+Enable our wind turbine technicians to keep track of their tasks.
+:::
+
+::::
+
+## Show Activity
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`hourglass;2.5em;sd-mr-1` Display Activity with Indicators
+:link: indicators_activity
+:link-type: doc
+
+Show activity with indicators just as rotating blades show the activity of wind turbines.
+:::
+
+:::{grid-item-card} {octicon}`versions;2.5em;sd-mr-1` Update Progressively
+:link: progressive_layouts
+:link-type: doc
+
+Unleash the power of generators and reactive expressions in our Wind Turbine Apps!
+:::
+
+:::{grid-item-card} {octicon}`image;2.5em;sd-mr-1` Build an Image Classifier
+:link: build_image_classifier
+:link-type: doc
+
+We will build an image classifier to identify wind turbine images.
+:::
+
+::::
+
+## Improve Performance
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`versions;2.5em;sd-mr-1` Add Caching
+:link: caching
+:link-type: doc
+
+Caching allows our apps to store and reuse valuable computations, reducing the energy required for calculations and making our apps run faster and smoother.
+:::
+
+:::{grid-item-card} {octicon}`kebab-horizontal;2.5em;sd-mr-1` Defer the Load
+:link: defer_load
+:link-type: doc
+
+Deferring the load will improve the performance perceived by the user
+:::
+
+::::
+
+## Periodic Updates
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`clock;2.5em;sd-mr-1` Build a Streaming Dashboard
+:link: build_streaming_dashboard
+:link-type: doc
+
+Build a live, updating dashboard to monitor your wind turbines.
+:::
+
+::::
+
+## Style your App
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`zap;2.5em;sd-mr-1` Apply A Design
+:link: design
+:link-type: doc
+
+Just as wind turbines come in various designs, this section empowers us to choose a design that best suits our app's style and functionality.
+:::
+
+:::{grid-item-card} {octicon}`paintbrush;2.5em;sd-mr-1` Use Styles
+:link: style
+:link-type: doc
+
+Fine-tune the styling of our components with `styles` and `stylesheets`. This will ensure our wind turbine data apps are both aesthetically pleasing and user-friendly.
+:::
+
+:::{grid-item-card} {octicon}`image;2.5em;sd-mr-1` Build a Polished Dashboard
```

Comment:
We should build a version of the *DataStore* dashboard that can be done with features that *basic* users would understand. You can help me if you do that @philippjfr .

Reply:
Done